docs/known_bugs.rst

```diff
@@ -19,82 +36,82 @@ Another way to get the up-to-date `scikit-image` version is::
 Although this solution takes much longer (~5 minutes) to install than the
 first solution.
 
-On some platforms, install fails due to `vispy`
------------------------------------------------
+On some platforms, install fails due to vispy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The 0.6.4 version of `vispy` fails to build for some combinations of
 platform/OS and Python versions.  `vispy` 0.6.5 has resolved this, but a
 workaround if you have an older version of vispy is to ensure you have a
-compatible version:
+compatible version::
 
-  % conda create -n jdaviz python=3.8
-  % conda activate jdaviz
-  % pip install vispy>=0.6.5
-  % pip install jdaviz --no-cache-dir
+    conda create -n jdaviz python=3.8
+    conda activate jdaviz
+    pip install vispy>=0.6.5
+    pip install jdaviz --no-cache-dir
 
 See `Issue #305 <https://github.com/spacetelescope/jdaviz/issues/305>`_ for
 updates on this topic.
 
-On some platforms, install fails due to `bottleneck`
-----------------------------------------------------
+On some platforms, install fails due to bottleneck
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 In a conda environment, where numpy was installed using conda, installing
 jdaviz using pip will attempt to pull bottleneck from PyPI. This might result
 in bottleneck trying to build numpy from source and crash, stalling the
 installation altogether. When this happens, exit the installation, install
 bottleneck with conda, and try to install jdaviz again.
 
-Collapse Plugin spectral bounds don't match selected region
------------------------------------------------------------
+In Python 3.7.1, install fails due to terminado
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Spectral bounds are off in the plugin compared to the spectrum viewer.
+In Python 3.7.1, installing jdaviz might produce this error::
 
+    ERROR: Cannot uninstall 'terminado'. It is a distutils installed project...
 
-Ghost subsets and models in spectrum viewer
```

Comment:
@camipacifici and @PatrickOgle , I removed mention of "ghost bug" as you requested.